src/cdktf-config.ts

```diff
@@ -149,6 +150,6 @@ export class CdktfConfig {
       },
     });
 
-    new ReadmeFile(project, "README.md", options);
+    new ReadmeFile(project, "README.md", { ...options });
```

Comment:
Hmm, I'm unsure why this change was required? Highlighting because maybe it's a left-over change from when you were overriding options here?